modules/aws-rds-integration/iam-roles/iam-roles.yaml

```
@@ -0,0 +1,79 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Deploys an IAM role allowing Common Fate to provision access to an RDS instance in the AWS account"
+Parameters:
+  RoleName:
+    Type: String
+    Description: Name for the IAM role
+    Default: common-fate-provision-rds
+  ExternalID:
+    Type: String
+    Description: The ExternalID to be used in the trust relationship
+  CommonFateAccount:
+    Type: String
+    Description: The AWS account ID that your Common Fate deployment is running in
+
+Metadata:
+  CF::Template: AWSAudit
```

These are intended to be used as metadata allowing us to introspect our own templates in the customer environment - the `CF::Template` field should be globally unique. We're already using `AWSAudit` on the audit role template, so we should name this something else

```suggestion
  CF::Template: AWSRDSProvisioner
```